src/screens/PinFlow/CreateTransactionPassword.tsx

```diff
@@ -60,29 +56,34 @@ export class CreateTransactionPassword extends PureComponent<Props, State> {
   render() {
     const { isVisible, password } = this.state;
     return (
-      <KeyboardAvoidingView style={styles.container} behavior="height">
+      <ScreenTemplate
+        footer={
+          <Button title="Save" onPress={this.onSave} disabled={password.length < CONST.transactionPasswordLength} />
```

Comment:
would be good to rename const `transactionPasswordLength` to `transactionPasswordMinLength`